bundles/model/org.eclipse.smarthome.model.script/src/org/eclipse/smarthome/model/script/scoping/ScriptImplicitlyImportedTypes.java

```diff
@@ -93,6 +95,9 @@
         result.add(Voice.class);
         result.add(ThingAction.class);
 
+        result.add(ImperialUnits.class);
```

Comment:
Could you please elaborate a bit on why `ImperialUnits` and `SIUnits` classes are added, but not `SmartHomeUnits` and `MetricPrefix`?

Reply:
`MetricPrefix` is simply missing here. `SIUnits` and `ImperialUnits` both extend `SmartHomeUnits`, so all units defined there are also in the scope of scripts/rues.

I will update the PR and add appropriate unit tests.